integration_test/external_tests/pages/settings_page.dart

```diff
@@ -82,6 +84,24 @@ Future<void> importAnAccountBySeeds(AppiumWebDriver driver, String accountType,
 
   await enterAccountAlias(driver, alias);
 
-  var continueButton = await driver.findElement(continueButtonLocator);
-  await continueButton.click();
+  var continueWithouItbutton = await driver.findElement(continueWithouItbuttonLocation);
```

Comment:
@ppupha Please manual first before update the source code in the common functions. It can make all script failed. Please check this step